keps/sig-node/127-user-namespaces/README.md

```diff
@@ -551,16 +589,20 @@ use container runtime versions that have the needed changes.
 
 ##### Beta
 
-- Make plans on whether, when, and how to enable by default
+- Gather and address feedback from the community
```

Comment:
We are still targeting another alpha as we add volume support here.

Reply:
Also, need a line about removing/changing restrictions on the what volumes will be allowed.

Reply:
> removing/changing restrictions on the what volumes will be allowed

Done